pkg/config/analysis/analyzers/testdata/injection.yaml

```diff
@@ -99,3 +99,17 @@ spec:
   containers:
   - image: gcr.io/google-samples/microservices-demo/adservice:v0.1.1
     name: server
+---
+# Pod that explicitly disables injection with label. Should not generate a warning.
+apiVersion: v1
+kind: Pod
+metadata:
+  name: podinjectiondisabled-label
+  namespace: default
+  labels:
+    sidecar.istio.io/inject: "false"
```

Comment:
do we support this as a label?  I thought `sidecar.istio.io/inject` was supported as an annotation ([doc](https://istio.io/latest/docs/reference/config/annotations/)).  In which case there is a test already covering the annotation on line 76

Reply:
@GregHanson See https://istio.io/latest/docs/setup/additional-setup/sidecar-injection/#controlling-the-injection-policy. If both are set, we prefer the label.

Reply:
@hanxiaop thanks for clarifying, I see the annotation and label logic in `/pkg/kube/inject/inject.go`.